docs/resources/samples/move-rows-across-tables.md

```diff
@@ -0,0 +1,256 @@
+---
+title: 'Move rows across tables in Office Scripts'
+description: 'Learn how to move rows across tables by saving filters, then processing and reapplying the filters.'
+ms.date: 03/30/2021
+localization_priority: Normal
+---
+
+# Move rows across tables by saving filters, then processing and reapplying the filters
+
+This script achieves the following:
+
+* Select rows from the source table where the value in a column is equal to _some value_.
+* Move all selected rows into another (target) table on another worksheet.
+* Reapply the relevant filters on the source table.
+
+![Screenshots of workbook before and after](../../images/table-filter-before-after.png)
+
+## Solution
+
+There are two script variants. The main difference is in the way the rows are selected.
+
+* In the [first variant](#move-rows-using-table-filter), the rows are selected by applying the table filter and reading the visible range.
+* In the [second](#move-rows-using-range-values), the rows are selected by reading the values and extracting the row values.
+
+## Sample Excel file
+
+Download the file <a href="input-table-filters.xlsx">input-table-filters.xlsx</a> used in this solution to try it out yourself!
+
+## Office Scripts
```

Comment:
Please see suggestion in previous file removing unnecessary navigation and heading levels.